src/redisearch_rs/inverted_index/src/lib.rs

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2006-Present, Redis Ltd.
+ * All rights reserved.
+ *
+ * Licensed under your choice of the Redis Source Available License 2.0
+ * (RSALv2); or (b) the Server Side Public License v1 (SSPLv1); or (c) the
+ * GNU Affero General Public License v3 (AGPLv3).
+*/
+
+use std::io::{Read, Seek, Write};
+
+pub use ffi::{RSIndexResult, t_docId};
+
+/// A delta is the difference between document IDs. It is mostly used to save space in the index
+/// because document IDs are usually sequential and the difference between them is small. With the
+/// help of encoding, we can store the difference (delta) efficiently instead of the full document
+/// ID.
```

Comment:
```suggestion
/// A delta is the difference between document IDs. It is mostly used to save space in the index
/// because document IDs are usually sequential and the difference between them are small. With the
/// help of encoding, we can optionally store the difference (delta) efficiently instead of the full document
/// ID.
```